logging/src/main/java/io/confluent/common/logging/LogRecordBuilder.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.common.logging;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaAndValue;
+import org.apache.kafka.connect.data.SchemaBuilder;
+
+public interface LogRecordBuilder<T> {
+  String FIELD_LOGGER = "logger";
+  String FIELD_LEVEL = "level";
+  String FIELD_TIME = "time";
+  String FIELD_MESSAGE = "message";
+
+  String NAMESPACE = "io.confluent.common.logging";
+
+  static SchemaBuilder baseSchemaBuilder() {
+    return SchemaBuilder.struct()
+        .name(NAMESPACE + "StructuredLogRecord")
```

Comment:
nit: either add a period between `NAMESPACE` and the name, or at the end of `NAMESPACE`? Everything else looks great. Sorry for the nit :)

Reply:
No worries - this is not a nit. Very important point in fact. Good catch!